builtin/credential/cert/path_login.go

```diff
@@ -127,6 +132,12 @@ func (b *backend) pathLoginRenew(
 		return nil, nil
 	}
 
+	policies := cert.Policies
+	sort.Strings(policies)
+	if strings.Join(policies, ",") != req.Auth.InternalData["policies"] {
```

Comment:
If I see reflect.DeepEqual in the code anywhere and if there is a possibility of removing it, in favor of joining and string comparison, I'll do it. :+1: